crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,243 @@
+# Crosswalk for Citation File Format 1.2.0
+
+## Format information
+
+Citation File Format (CFF) is a YAML format for providing citation metdaata for software (and experimentally for datasets).
+
+- Website: <https://citation-file-format.github.io/>
+- Specifications: <https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema-guide.md>
+- Schema: <https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema.json> (JSON Schema)
+
+## Crosswalk specifics
+
+The following sections provide details for directional mapping 
+in cases where mapping is not bidirectionally unambiguous.
+
+### `downloadUrl` <=> `repository-artifact` (and potentially others)
+
+- CodeMeta `downloadUrl`: "If the file can be downloaded, URL to download the binary."
+- CFF `repository-artifact`: "The URL of the work in a build artifact/binary repository."
+
+While the scope of `repository-artifact` is documented to be somewhat narrower than `downloadUrl`, 
+its main use case is discovery (e.g. for reuse and reproducibility),
+which is arguably the same as for `downloadUrl`.
+Therefore the two can be assumed to signify the same.
+
+#### CFF to CodeMeta
+
+In some cases, values for the following CFF fields could also be mappable to CodeMeta `downloadUrl`:
+
+- `url`, when the URL points to a download URL
+- `repository`, when the repository is containing a downloadable artifact of the software
+- `identifiers[i].type==url`, when the identifier resolves to a downloadable artifact of the software
+
+Future versions of CFF may document this more clearly (<https://github.com/citation-file-format/citation-file-format/issues/326>).
+
+### `installUrl` <=> different potential mapping targets
```

Comment:
drop `installUrl` because it might be repository artifact, but it is already crossawlked with `downloadUrl`

Reply:
That's one of the issues though, isn't it, with crosswalks in general? They assume that there is a 1:1 relation between properties, whereas in reality (or perhaps due to CFF schema design), there is sometimes a 1:n, sometimes a n:1, sometimes a n:m relation. E.g., here, `installUrl` could be mapped to from `url` or `repository-artifact`...

Reply:
Removed the `installUrl` section from the CSV with a note in the README in https://github.com/codemeta/codemeta/pull/263/commits/44a286e17fee95fd19886c27eb5eb17df7d2215d.

However, I still think that this points to some larger issues with the crosswalks relation as mentioned above. Not something that can be resolved in this PR though.